### DISCUSSION TITLE
Contributions Widget not working

### DISCUSSION THREAD

**Discussion by gaurav7902** [community/community#145496] [Mobile]
In Funtouch OS , widget is added but doesn't refresh automatically and even after refreshing manually the contributions don't appear.
Needs to again add the widget to see the latest contributions.

**Comment by iskae** [comment]
Hi @gaurav7902 👋 

Thank you so much for reaching out ❤️ We are relying on Android callbacks for updating the widget periodically. Maybe our requests for updating the widget are evaluated with lower priority on Funtouch OS 😬  
I have created an internal issue for investigating it and if possible, fixing it 🙇 

Thanks once again and please stay tuned for our future releases ❤️

**Comment by Enderteck** [comment]
![Screenshot_20241226_161658_One UI Home.jpg](https://github.com/user-attachments/assets/4b364158-458c-4fd0-9f3d-22d01adf2b70)


This is what the contributions widget shows on my home screen.

**Comment by yogeshpaliyal** [comment]
I am not able to add the widget kn Mi Launcher , here is the error stack trace, but able to add widgets for others apps and that are working as expected.

![Screenshot_2025-01-01-23-34-37-872_com.miui.bugreport.jpg](https://github.com/user-attachments/assets/72192156-8ae1-4877-ac6c-2fe6498d3b78)

**Comment by andjsch** [comment]
Same here with a Pixel. I see the contributions but they won't auto update. I have to edit the widget and tap on my account again. Resizing the widget is not enough to make it update.
